elasticsearch/tests/elasticsearch/elasticsearch.tests.bom

```diff
@@ -41,6 +41,7 @@ brooklyn.catalog:
             id: elasticsearch-cluster
             brooklyn.config:
               cluster.initial.size: 1
+              elasticsearch.bind.address: _site_
```

Comment:
could you add (below) instead as it will fail fast
```
brooklyn.parameters:
- name: elasticsearch.bind.address
  type: string
  constraints:
  - required
```